diff --git a/node_modules/@tendermint/belt/index.js b/node_modules/@tendermint/belt/index.js
index e6d12ff..3d21adc 100644
--- a/node_modules/@tendermint/belt/index.js
+++ b/node_modules/@tendermint/belt/index.js
@@ -1,4 +1,9 @@
 let node = false;
+// if globalThis exists and is not window, use it
+if (typeof globalThis !== 'undefined' &&  typeof window === 'undefined') {
+   self.window = globalThis;
+}
+
 try {
     node = (Object.prototype.toString.call(global.process) === '[object process]');
 }
diff --git a/node_modules/@tendermint/belt/src/bytes.ts b/node_modules/@tendermint/belt/src/bytes.ts
index c19d880..6491611 100644
--- a/node_modules/@tendermint/belt/src/bytes.ts
+++ b/node_modules/@tendermint/belt/src/bytes.ts
@@ -1,5 +1,6 @@
 import { Bytes, USVString } from '@tendermint/types';
 
+
 let TextDecoder: { new (): { decode (bytes: Bytes): USVString } };
 let TextEncoder: { new (): { encode (string: USVString): Bytes } };
 
