diff --git a/node_modules/@tendermint/sig/index.js b/node_modules/@tendermint/sig/index.js
index e6d12ff..a13b71c 100644
--- a/node_modules/@tendermint/sig/index.js
+++ b/node_modules/@tendermint/sig/index.js
@@ -1,4 +1,8 @@
 let node = false;
+// if globalThis exists and is not window, use it
+if (typeof globalThis !== 'undefined' &&  typeof window === 'undefined') {
+    self.window = globalThis;
+}
 try {
     node = (Object.prototype.toString.call(global.process) === '[object process]');
 }
